wurst/systems/spawns/ResourceSpawns.wurst

```diff
@@ -164,59 +229,114 @@ function initItemSpawnInfo()
 
 function adjustBaseSpawnrates()
     //Replacement for modStats function of the legacy system.
-    //Dynamically adjust overall item & animal amounts (and specific item type amounts but that is handled by the itemSpawnInfo class now)
+    //Dynamically adjust overall item & animal amounts (and specific item type amounts but thats handles by spawnInfo classes now)
     udg_ITEM_BASE = max(.15,udg_ITEM_BASE-0.2)
     udg_FOOD_BASE = max(.15,udg_FOOD_BASE-0.2)
 
-    // This part actually doesnt do anythin since udg_BADDIE_BASE is always 1.00 - Keeping it for now to be safe
-    udg_PANTHER_RATE = R2I(1*udg_BADDIE_BASE)
-    udg_BEAR_RATE = R2I(1*udg_BADDIE_BASE)
-    udg_SNAKE_RATE = R2I(1*udg_BADDIE_BASE)
-    udg_WOLF_RATE = R2I(2*udg_BADDIE_BASE)
+function spawnFishOrHawk(rect spawnRect)
+    if udg_ITEM_LIMIT_MODE and udg_FISH_CURRENT < udg_FISH_MAX
+        let rand = GetRandomInt(1, 13)
+        let spawnPoint = spawnRect.randomPoint()
+        if rand <= 4
+            new Hawk(spawnPoint)
+        else if rand <= 6
+            new GreenFish(spawnPoint)
+        else
+            new Fish(spawnPoint)
+        udg_FISH_CURRENT += 1
+
+function spawnFishAndHawks()
+    doPeriodicallyCounted(ANIMATION_PERIOD * 2, FISH_PER_AREA) fishHawkLoop ->
+        spawnFishOrHawk(gg_rct_out_1_1)
+        spawnFishOrHawk(gg_rct_out_3_2)
+        spawnFishOrHawk(gg_rct_out_1_2)
+        spawnFishOrHawk(gg_rct_out_5_2)
+        spawnFishOrHawk(gg_rct_out_2_2)
+        spawnFishOrHawk(gg_rct_out_3_1)
+        spawnFishOrHawk(gg_rct_out_4_1)
+        spawnFishOrHawk(gg_rct_out_4_2)
+        spawnFishOrHawk(gg_rct_our_5_1)
+        spawnFishOrHawk(gg_rct_out_2_1)
+        spawnFishOrHawk(gg_rct_fish_new_2)
+        spawnFishOrHawk(gg_rct_fish_new_3)
+        spawnFishOrHawk(gg_rct_fish_new_1)
+        spawnFishOrHawk(gg_rct_fish_new_4)
+        spawnFishOrHawk(gg_rct_fish_new_6)
+        spawnFishOrHawk(gg_rct_fish_new_5)
+        spawnFishOrHawk(gg_rct_fish_new_7)
+        spawnFishOrHawk(gg_rct_fish_new_8)
 
 init
     initIslandWeights()
     initSpawnRegionWeights()
     initItemSpawnInfo()
+    initAnimalSpawnInfo()
     initIslandSpawners()
     registerGameStartEvent() ->
         gameStartTime = getElapsedGameTime()
 
-        //First spawns. Do twice to retain original amounts after  halving the items per spawn and doubling cycle frequency.
-        handleItemSpawning()
-        doAfter(10) ->
-            handleItemSpawning()
+        //First spawns. Do twice to retain original amounts after halving the items per spawn and doubling cycle frequency.
+        handleAnimalSpawning()
+        doAfter(5) ->
+            handleAnimalSpawning()
+            doAfter(5) ->
+                handleItemSpawning()
+                doAfter(5) ->
+                    handleItemSpawning()
+                    doAfter(5) ->
+                        spawnFishAndHawks()
+                        doAfter(5) ->
+                            addItemsToBushes()
```

Comment:
This seems a bit weird haha

Reply:
I didnt come up with any other idea how to space these out

Reply:
Perhaps

```
doAfter(0, -> handleAnimalSpawning())
doAfter(5, -> handleAnimalSpawning())
doAfter(10, -> handleItemSpawning())
doAfter(15, -> handleItemSpawning())
doAfter(20, -> spawnFishAndHawks())
doAfter(25, -> addItemsToBushes())
```